roles/downloadmq/tasks/main.yml

```
@@ -2,33 +2,38 @@
 - name: Including common vars for each platform
   ansible.builtin.include_vars: "common_{{ role_name }}.yml"
 
-# Set release types based on version mod, useful for different naming conventions
-- name: Set release type to LTS
+# Set convention types based on release version (new for >9.3 releases)
+- name: Set convention type to old for 9.3 and 9.2 LTS
   ansible.builtin.set_fact:
-    release: lts
-  when: version%10 == 0 
+    convention: old
+  when: version==930 or version==920
 
-- name: Set release type to CD
+- name: Set convention type to new to other releases
   ansible.builtin.set_fact:
-    release: cd
-  when: version%10 != 0 
+    convention: new
+  when: not convention
```

I got a 'convention is not defined' error here - changing to the below fixed it

```suggestion
  when: convention is undefined
```